src/components/UserSelector.tsx

```diff
@@ -72,11 +68,9 @@ export const UserSelector: React.FC<Props> = ({
             <a
               key={user.id}
               href={`#user-${user.id}`}
-              onClick={() => {
-                onChange(user);
-              }}
+              onClick={() => selectAuthor(user)}
```

Comment:
It's not recommended to use inline functions in props if they can be avoided. In this case, you can define a function outside of the component's body that calls selectAuthor with a user argument and pass that function as a prop. This way, a new function won't be created on each render.